src/runtime/pkg/katautils/config.go

```diff
@@ -57,6 +57,10 @@ const (
 
 	// the maximum amount of PCI bridges that can be cold plugged in a VM
 	maxPCIBridges uint32 = 5
+	// For mor info why these values, see:
```

Comment:
```suggestion
	// For more info on why these values were chosen, see:
```

Reply:
@zvonkok This may just be a problem with the GitHub UI, but I don't see that fix applied, yet the comment is marked as resolved. Can you check if the typo is fixed?

Reply:
@c3d Done